content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -842,6 +842,13 @@ Kubernetes marks a Deployment as _progressing_ when one of the following tasks i
 * The Deployment is scaling down its older ReplicaSet(s).
 * New Pods become ready or available (ready for at least [MinReadySeconds](#min-ready-seconds)).
 
+When the rollout becomes progressing, the Deployment controller adds a DeploymentCondition with the following
```

Comment:
```suggestion
When the rollout becomes “progressing”, the Deployment controller adds a condition with
the following
```
`DeploymentCondition` isn't part of the Kubernetes API; it's an implementation detail of the Golang implementation.
_In theory_ someone could drop in their own conformant Deployment controller and use that. We should write docs that are still relevant for that case.